plugins/scaffolder-backend-module-confluence-to-markdown/src/actions/confluence/helpers.ts

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Config } from '@backstage/config';
+import { ResponseError, ConflictError, InputError } from '@backstage/errors';
+import fs from 'fs-extra';
+import fetch, { Response } from 'node-fetch';
+
+interface Links {
+  webui: string;
+  download: string;
+  thumbnail: string;
+  self: string;
+}
+
+interface Metadata {
+  mediaType: string;
+}
+
+export interface Result {
+  id: string;
+  type: string;
+  status: string;
+  title: string;
+  metadata: Metadata;
+  _links: Links;
+}
+
+export interface Results {
+  results: Result[];
+}
+
+export const readFileAsString = async (fileDir: string) => {
+  const content = await fs.readFile(fileDir, 'utf-8');
+  return content.toString();
+};
+
+export const fetchConfluence = async (relativeUrl: string, config: Config) => {
+  const baseUrl = config.getString('confluence.baseUrl');
+  const token = config.getString('confluence.token');
+  const response: Response = await fetch(`${baseUrl}${relativeUrl}`, {
+    method: 'GET',
+    headers: {
+      Authorization: `Bearer ${token}`,
+    },
+  });
+  if (!response.ok) {
+    throw await ResponseError.fromResponse(response);
+  }
+
+  return response.json() as Promise<any>;
```

Comment:
```suggestion
  return response.json();
```

I believe this cast is redundant